### PR TITLE
data from APM and FES

### PR DESCRIPTION
**Pull request by topepo**
More data sets for modeling from books. Still about 3-4MB of space left in the package.

### CODE REVIEW

**Review by simonpcouch — COMMENTED** [review]
An initial pass; haven't yet pulled these down locally.

File: R/chem_proc_yeild.R [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#' Chemical manufacturing process data set
+#'
+#' @description
+#' A data set that models yield as a function of biological material predictors
+#' and chemical structure predictors.
+#'
+#' @name chem_proc_yeild
```
[the remark this answers is not in the document]
Comment (topepo) [reply R/chem_proc_yeild.R:7]: 🙄 👍

File: data-raw/ischemic_stroke.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# ------------------------------------------------------------------------------
+
+tidymodels_prefer()
+theme_set(theme_bw())
+options(pillar.advice = FALSE, pillar.min_title_chars = Inf)
+
+# ------------------------------------------------------------------------------
+
+load(url("https://github.com/topepo/FES/raw/master/Data_Sets/Ischemic_Stroke/stroke_data.RData"))
```
Comment (simonpcouch) [inline data-raw/ischemic_stroke.R:13]:
```suggestion
load(url("https://github.com/topepo/FES/raw/06812c48a21882808403cee338b8312fdbd35a46/Data_Sets/Ischemic_Stroke/stroke_data.RData"))
```

Let's use a permalink here!

File: data-raw/leaf_id_flavia.R [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# ------------------------------------------------------------------------------
+
+tidymodels_prefer()
+theme_set(theme_bw())
+options(pillar.advice = FALSE, pillar.min_title_chars = Inf)
+
+# ------------------------------------------------------------------------------
+
+flavia_url <-
+  "https://github.com/SMART-Research/leaffeatures_paper/raw/main/data_all_with_label_flavia_with_species.csv"
```
Comment (simonpcouch) [inline data-raw/leaf_id_flavia.R:14]:
```suggestion
  "https://github.com/SMART-Research/leaffeatures_paper/raw/65ffd8c8b926b8df3f499c9224d6073975db5c3c/data_all_with_label_flavia_with_species.csv"
```

Same thought here.

**Review by simonpcouch — APPROVED** [review]
Looks great. :)

**Pull request merged**